enhancements/storage/csi-vsphere-operator.md

```diff
@@ -0,0 +1,172 @@
+---
+title: csi-ebs-operator
+authors:
+  - "@gnufied"
+reviewers:
+  - "@jsafrane”
+  - "@fbertina"
+  - "@chuffman"
+approvers:
+  - "@..."
+creation-date: 2020-10-05
+last-updated: 2020-10-05
+status: implementable
+see-also: https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-driver-install.md
+replaces:
+superseded-by:
+---
+
+# CSI Driver operator for vSphere
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes deployment of vSphere CSI driver on Openshift as a default component.
+
+## Motivation
+
+* vSphere is a key cloud provider for OpenShift and is supported in Openshift 3.x and 4.x. We need it to be supported and available even if in-tree drivers have been removed.
+* vSphere CSI driver provides new features such as - volume expansion, snapshotting and cloning which were previously unavailable with intree driver.
+
+### Goals
+
+* Create an operator to install the vSphere CSI driver.
+* Publish the driver and operator via default OCP build pipeline.
+* Enable creation of vSphere CSI storageclass, so as OCP users can start consuming vSphere CSI volumes without requiring further configuration.
+
+### Non-Goals
+
+* We don't intend to create a brand new driver with this KEP but merely want to deploy and configure upstream vSphere CSI driver - https://github.com/kubernetes-sigs/vsphere-csi-driver/
+
+
+## Proposal
+
+OCP ships with a vsphere-csi-driver-operator by default which is managed by [cluster-storage-operator](https://github.com/openshift/cluster-storage-operator/).
```

Comment:
Does it make sense to call it `vmware-vsphere-csi-driver-operator`?

As in `aws-ebs-csi-driver-operator`, `gcp-pd-csi-driver-operator`, `openstack-cinder-csi-driver-operator`?

Reply:
sure. lets take the naming discussion in operator PR?

Reply:
There's no operator PR - you need to ask for openshift/ repo first. And for the repo I'd appreciate to use the operator name.

Reply:
renamed